python/Ganga/Core/GangaRepository/SubJobXMLList.py

```diff
@@ -517,3 +495,45 @@ def _private_display(self, reg_slice, this_format, default_width, markup):
             ds += markup(this_format % tuple(vals), colour)
 
         return ds
+
+    @staticmethod
+    def JobHasChildrenTest(jobDirectory):
+        """ Return True/False if given (job?) object has children associated with it
+        Args:
+            jobDirectory (str): name of folder to be examined
+        """
+
+        count = SubJobXMLList.countSubJobDirs(jobDirectory)
+        if count:
+            return True
+        else:
+            return False
```

Comment:
This could just be:
```python
@staticmethod
def jobHasChildrenTest(jobDirectory):
    return bool(SubJobXMLList.countSubJobDirs(jobDirectory))
```
at which point the one place it's used may as well just be:
```python
has_children = has_children or SubJobXMLList.countSubJobDirs(path.dirname(fn))
```
and this function is not needed.